lib/SimplePie/SimplePie.php

```diff
@@ -1514,71 +1534,58 @@ protected function fetch_data(&$cache)
 					}
 				}
 				// Check if the cache has been updated
-				else //if ($cache->mtime() + $this->cache_duration < time())	//FreshRSS removed
+				else
```

Comment:
The code below looks like a big change, but it is not: is is mainly some comments and indentation changed.